crates/turborepo-lib/src/run/mod.rs

```diff
@@ -273,6 +273,7 @@ impl Run {
         };
 
         // if we are forcing the daemon, we don't want to fallback to local discovery
+        //#[cfg(feature = "daemon-package-discovery")]
         let (fallback, duration) = if let Some(true) = self.opts.run_opts.daemon {
```

Comment:
If we're constructing `fallback`, we should make sure to use it. The call `LocalPackageDiscoveryBuilder.build()` can result in a `yarn --version` or `pnpm --version` if the user hasn't specified a `packageManager` in `package.json`.

Reply:
Ah, ok. I was trying to decide how much I needed to cut. Seems like cut all of the fallback stuff. I'll see what I can do.